db-tests/src/test/java/gov/nasa/jpl/aerie/database/ExternalEventTests.java

```diff
@@ -0,0 +1,2168 @@
+package gov.nasa.jpl.aerie.database;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.postgresql.util.PSQLException;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@SuppressWarnings("SqlSourceToSinkFlow")
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class ExternalEventTests {
+
+  //region Generic Database Testing Setup
+  private DatabaseTestHelper helper;
+  private MerlinDatabaseTestHelper merlinHelper;
+  private Connection connection;
+
+  @AfterEach
+  void afterEach() throws SQLException {
+    helper.clearSchema("merlin");
+  }
+
+  @BeforeAll
+  void beforeAll() throws SQLException, IOException, InterruptedException {
+    helper = new DatabaseTestHelper("external_event", "Activity Directive Changelog Tests");
+    connection = helper.connection();
+    merlinHelper = new MerlinDatabaseTestHelper(connection);
+  }
+
+  @AfterAll
+  void afterAll() throws SQLException, IOException, InterruptedException {
+    helper.clearSchema("merlin");
+    helper.close();
+    connection = null;
+    helper = null;
+  }
+  //endregion
+
+
+  //region Commonly Repeated Variables
+  // Source Type (st)
+  final String st = "Test";
+
+  // Derivation Group (dg)
+  final String dg = "Test Default";
+
+  // Event Type (et)
+  final String et = "Test";
+
+  // Created At (ca)
+  final String ca = "2024-01-01T00:00:00Z";
```

Comment:
Should've commented on this sooner, but the fact that these ought to be `static` made me think of it: two Aerie-specific stylings these constant names don't obey:

1) Constants are in CAPS_CASE
2) Constant names shouldn't be two-letter abbreviations.

So they should be named like so (removing the comments that explain the abbreviations, as they become redundant, and adding `static`)
```suggestion
  final static String SOURCE_TYPE = "Test";
  final static String DERIVATION_GROUP = "Test Default";
  final static String EVENT_TYPE = "Test";
  final static String CREATED_AT = "2024-01-01T00:00:00Z";
```